modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -1,14 +1,10 @@
 // Module included in the following assemblies:
 //
-// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa-architecture-models.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="rosa-hcp-classic-comparison_{context}"]
-= Comparing ROSA with hosted control planes and ROSA Classic
-
-{hcp-title-first} offers a different way to create a managed {product-title} (ROSA) cluster. {hcp-title} offers a reduced-cost solution with focuses on reliability and efficiency. With a focus on efficiency, you can quickly create a new cluster and deploy applications in minutes.
-
-{hcp-title} requires only a minimum of two nodes making it ideal for smaller projects while still being able to scale to support larger projects and enterprises.
+= Comparing ROSA with HCP and ROSA Classic
 
 .ROSA architectures comparison table
```

Comment:
Let me share a revised comparison table: https://docs.google.com/document/d/161mYQMA_a5zOOzluRUi7V1ywPZ1mVBC2IZrRF_e6t6w/edit 

There are multiple issues with that:

1. We can skip the regional availability in the table. I suggest inserting a statement to refer to the linked AWS Documentation for regional availability. The linked AWS Documentation actually lists both HCP and Classic regional availability. Right now, we are listing the regions for HCP and linking the document for Classic - something not ideal.
2. Deployment - Please remove the difference where about CLI only is supported in HCP. OCM Console/UI also now supports creation of ROSA HCP so there's no difference in that any more. 
3. Architecture - Please remove "Customer can access control plane infrastructure through dedicated and explicitly exposed endpoints" because this is not coming clear. Instead suggest writing as 3rd point so first 2 points match agains each column.